### PR TITLE
003_Check Palindrome.

### PR DESCRIPTION
**Pull request by dmurugesh**

### CODE REVIEW

File: jsbasics/palindrome.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+let readline = require("readline-sync");
+
+let str = readline.question("enter word: ");    // use to enter user input word
+
+const palindrome = str =>
```
Comment (kuism) [inline jsbasics/palindrome.js]: not aligned properly

File: jsbasics/palindrome.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+let readline = require("readline-sync");
+let str = readline.question("enter word: ");    // use to enter user input word
+
+const palindrome = str => {
+  str = str.toLowerCase().replace(/[^a-z]+/g, "");    // tolowercase - use to convert into lowerstring., replace() - return matches of a pattern replaced by a replacement
+  if (str === str.split("").reverse().join("")) {     //  ,"" remove spaces, /[^a-z] - convert the string into lowercase,
```
Comment (kuism) [inline jsbasics/palindrome.js:6]: you should do this without using string functions

Comment (dmurugesh) [reply jsbasics/palindrome.js:6]: In today's call, you mentioned if you're ok with string functions you can use string functions. Could you please assist me is this necessary to change this program.

File: jsbasics/palindrome.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+let readline = require("readline-sync");
```
Comment (NidhitheCoder) [inline jsbasics/palindrome.js]: better to use const here.

**Pull request merged**